### PR TITLE
installer/windows: add build script with instructions

### PR DESCRIPTION
**Pull request by dominic**

### CODE REVIEW

**Comment by dominic** [comment]
PTAL @tessr @jeffomatic

File: installer/windows/README.md [hunk 1]
```diff
@@ -43,7 +42,34 @@ The VC++ Redistributable can be downloaded from https://www.microsoft.com/en-us/
...
+## Building the Windows installer
+
+### Using the build script
+
+Create a copy of the sample batch file `buildChainExe.bat.sample` inside the
+`installer/windows` directory. In this case, we'll call it `buildChainExe.bat`.
+
+Open `buildChainExe.bat` in your editor of choice and make the following changes:
+
+* Replace all instances of `Z:\PATH\TO\CERT` with the path to your `.pfx`
```
Comment (tessr) [inline installer/windows/README.md]: Can we use environment variables here?

File: installer/windows/README.md [hunk 2]
```diff
@@ -43,7 +42,34 @@ The VC++ Redistributable can be downloaded from https://www.microsoft.com/en-us/
...
+
+```
+buildChainExe.bat
+```
+
+The resulting binary will be available at `ChainBundle/Chain_Core_[your-version].exe`.
+
+### Manual
+
+🚧 Hang on. Only go down here if you know what you're doing 🚧
```
Comment (tessr) [inline installer/windows/README.md]: LOL. But also maybe make it clear that this is just the same thing as the script.

File: installer/windows/buildChainExe.bat.sample [hunk 3]
```diff
@@ -0,0 +1,14 @@
+signtool sign -v -f Z:\PATH\TO\CERT -p CERT_PASSWORD cored.exe
+signtool sign -v -f Z:\PATH\TO\CERT -p CERT_PASSWORD ChainMgr/ChainMgr.exe
+cd ChainPackage
+candle -ext WixHttpExtension -ext WixUtilExtension ChainCoreInstaller.wxs
+light -ext WixHttpExtension -ext WixUtilExtension ChainCoreInstaller.wixobj
+signtool sign -v -f Z:\PATH\TO\CERT -p CERT_PASSWORD cab1.cab
+cd ../ChainBundle
+candle Bundle.wxs -arch x64 -ext WixBalExtension -dChainPackage.TargetPath=C:\code\src\chain\installer\windows\ChainPackage\ChainCoreInstaller.msi -dPostgresPackage.TargetPath=C:\code\src\chain\installer\windows\Postgres\postgresql-9.5.5-1-windows-x64.exe -dVCRPackage.TargetPath=C:\code\src\chain\installer\windows\Postgres\vcredist_x64.exe
```
Comment (tessr) [inline installer/windows/buildChainExe.bat.sample]: I wonder if we should also make the paths to the `.exe`s configurable.

File: installer/windows/buildChainExe.bat.sample [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+light Bundle.wixobj -ext WixBalExtension
+signtool sign -v -f Z:\PATH\TO\CERT
```
Comment (tessr) [inline installer/windows/buildChainExe.bat.sample]: I think this line needs to be removed

**Comment by jeffomatic** [comment]
Maybe we could take this one step further and commit an actual .bat file.

**Comment by tessr** [comment]
This looks awesome. I'm looking forward to using this :bat:

**Pull request merged**